src/figuresClasses.ts

```diff
@@ -1,19 +1,78 @@
-export interface Figure {
+  type Shape = 'triangle' | 'circle' | 'rectangle';
+  type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea: () => number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  public shape = Shape.Triangle;
```

Comment:
where do you have Shape.Triangle? I think you don't have such an object 😉 
you can write it the same as you have in other figures - `shape: Figure['shape'] = 'triangle';`

Reply:
Fixed